cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/InactiveOperationInfo.java

```diff
@@ -21,11 +21,14 @@
 
 import java.util.List;
 
-final class InactiveOperationInfo extends OperationInfo {
+/**
+ * OperationInfo representing an inactive operation.
+ */
+public final class InactiveOperationInfo extends OperationInfo {
```

Comment:
I see that the test classes are not part of the same namespace as code they test. Can you move the test classes to code namespace so that these classes can remain package-private?